src/sql/workbench/services/connection/browser/connectionManagementService.ts

```diff
@@ -780,9 +780,9 @@ export class ConnectionManagementService extends Disposable implements IConnecti
 			return true;
 		}
 		let azureResource = this.getAzureResourceForConnection(connection);
-		let accounts = await this._accountManagementService.getAccountsForProvider('azurePublicCloud');
+		let accounts = (await this._accountManagementService.getAccounts()).filter(a => a.key.providerId.startsWith('azure'));
```

Comment:
Our connection system is currently only working with Azure, this is one of the things that'll have to change once we get a proper account picker working.